extensions/mssql/src/hdfs/webhdfs.ts

```diff
@@ -465,6 +466,27 @@ export class WebHDFS {
 		});
 	}
 
+	/**
+	 * Get ACL status for given path
+	 * @param path The path to the file/folder to get the status of
+	 * @param callback Callback to handle the response
+	 * @returns void
+	 */
+	public getMounts(callback: (error: HdfsError, mounts: Mount[]) => void): void {
+		let endpoint = this.getOperationEndpoint('listmounts', '');
+		this.sendRequest('GET', endpoint, undefined, (error, response) => {
+			if (!callback) { return; }
+			if (error) {
+				callback(error, undefined);
+			} else if (response.body.hasOwnProperty('Mounts')) {
```

Comment:
will it always be capitalized?

Reply:
That's the official contract, yeah. I can try to check both but we're not doing this elsewhere